app/src/molecules/InstrumentCard/index.tsx

```diff
@@ -69,8 +69,18 @@ export function InstrumentCard(props: InstrumentCardProps): JSX.Element {
       {...styleProps}
     >
       {isGripperAttached ? (
-        <Flex justifyContent={JUSTIFY_CENTER} size="3.75rem">
-          <img src={flexGripper} alt="flex gripper" />
+        <Flex
+          justifyContent={JUSTIFY_CENTER}
+          size="3.75rem"
+          width={'60px'}
+          height={'54px'}
```

Comment:
the size is unnecessary now. also, maybe convert the pixels into rem? We typically try to use rem unless you are sizing an img html tag

```suggestion
          width="60px"
          height="54px"
```